codes/kotlin/chapter_array_and_linkedlist/array.kt

```diff
@@ -0,0 +1,101 @@
+/**
+ * File: array.kt
+ * Created Time: 2024-01-25
+ * Author: curtishd(1023632660@qq.com)
```

Comment:
curtishd (1023632660@qq.com)

(Add a space between the username and email)

Reply:
alright